egs/wsj/s5/steps/nnet3/decode_score_fusion.sh

```diff
@@ -110,6 +110,22 @@ if [ ! -z "$online_ivector_dir" ]; then
     ivector_opts="--online-ivectors=scp:$online_ivector_dir/ivector_online.scp --online-ivector-period=$ivector_period"
 fi
 
+# assign frame_subsampling_factor automatically if empty
+if [ -z $frame_subsampling_factor ]; then
+   frame_subsampling_factor=`cat ${model_dirs[0]}/frame_subsampling_factor` || exit 1;
+fi
+
+# check if standard chain system or not.
+if [ $frame_subsampling_factor -e 3 ]; then
```

Comment:
I think you mean -eq here.  And below, instead of -ne you should use !=, because they are non-integer.  Please test!